MYSETUP.md

```diff
@@ -0,0 +1,5 @@
+# Setup Process
```

Comment:
We should put the setup instructions in their own PR so that we keep the uploadSchools/index.js fix by itself

Reply:
is PR pull request?

Reply:
Yeah, PR is Pull Request